src/database/services/google-api-service.ts

```diff
@@ -0,0 +1,490 @@
+import { drive_v3, google } from "googleapis";
```

Comment:
This needs to be split into several files, currently it handles several different Google services including files, drive, translation and besides those it seems to also handle communication with OpenAPI services.